tdrs-backend/tdpservice/parsers/schema_defs/tanf/t1.py

```diff
@@ -1,114 +1,280 @@
-                    required=True, validators=[]),
-              Field(item="26AII", name='WORK_REQ_SANCTION', type='number', startIndex=96, endIndex=97,
-                    required=True, validators=[]),
-              Field(item="26AIII", name='FAMILY_SANC_ADULT', type='number', startIndex=97, endIndex=98,
-                    required=True, validators=[]),
-              Field(item="26AIV", name='SANC_TEEN_PARENT', type='number', startIndex=98, endIndex=99,
-                    required=True, validators=[]),
-              Field(item="26AV", name='NON_COOPERATION_CSE', type='number', startIndex=99, endIndex=100,
-                    required=True, validators=[]),
-              Field(item="26AVI", name='FAILURE_TO_COMPLY', type='number', startIndex=100, endIndex=101,
-                    required=True, validators=[]),
-              Field(item="26AVII", name='OTHER_SANCTION', type='number', startIndex=101, endIndex=102,
-                    required=True, validators=[]),
-              Field(item="26B", name='RECOUPMENT_PRIOR_OVRPMT', type='number', startIndex=102, endIndex=106,
-                    required=True, validators=[]),
-              Field(item="26CI", name='OTHER_TOTAL_REDUCTIONS', type='number', startIndex=106, endIndex=110,
-                    required=True, validators=[]),
-              Field(item="26CII", name='FAMILY_CAP', type='number', startIndex=110, endIndex=111,
-                    required=True, validators=[]),
-              Field(item="26CIII", name='REDUCTIONS_ON_RECEIPTS', type='number', startIndex=111, endIndex=112,
-                    required=True, validators=[]),
-              Field(item="26CIV", name='OTHER_NON_SANCTION', type='number', startIndex=112, endIndex=113,
-                    required=True, validators=[]),
-              Field(item="27", name='WAIVER_EVAL_CONTROL_GRPS', type='number', startIndex=113, endIndex=114,
-                    required=True, validators=[]),
-              Field(item="28", name='FAMILY_EXEMPT_TIME_LIMITS', type='number', startIndex=114, endIndex=116,
-                    required=True, validators=[]),
-              Field(item="29", name='FAMILY_NEW_CHILD', type='number', startIndex=116, endIndex=117,
-                    required=True, validators=[]),
-              Field(item="", name='BLANK', type='string', startIndex=117, endIndex=156, required=False, validators=[]),
-          ],
-        )
-      ]
-  )
+t1 = SchemaManager(schemas=[
+      RowSchema(
+            model=TANF_T1,
+            preparsing_validators=[
+                  validators.hasLength(156),
+            ],
+            postparsing_validators=[
+                  validators.if_then_validator(
+                        condition_field='RECEIVES_FOOD_STAMPS', condition_function=validators.matches(1),
+                        result_field='AMT_FOOD_STAMP_ASSISTANCE', result_function=validators.isLargerThan(0),
+                  ),
+                  validators.if_then_validator(
```

Comment:
this check can be dropped. In the new instruction, receipt status will be inferred from the amount.